server/methods/answers.js

```diff
@@ -1,186 +1,373 @@
 Meteor.methods({
-  createAnswer(answerData) {
-    var { description, questionId } = answerData;
-
-    var answer = {
-      ownerId: Meteor.userId(),
-      description: description,
-      upvotes: [],
-      downvotes: [],
-      createdAt: Date.now()
+    createAnswer(answerData) {
+        var {
+            description,
+            questionId,
+            questionOwnerEmail,
```

Comment:
as said in a previous review .. I don't think you need additional data from the client but `description` and `questionId`